ambry-api/src/main/java/com.github.ambry/config/CloudConfig.java

```diff
@@ -24,6 +24,16 @@
   public static final String CLOUD_DESTINATION_FACTORY_CLASS = "cloud.destination.factory.class";
   public static final String DEFAULT_CLOUD_DESTINATION_FACTORY_CLASS =
       "com.github.ambry.cloud.azure.AzureCloudDestinationFactory";
+  public static final String CRYPTO_SERVICE_FACTORY_CLASS = "vcr.crypto.service.factory.class";
```

Comment:
Are crypto and kms service factories already defined in another config which we can pass to the CloudBackupManager instead of duplicating them?  Will we ever have different factories used for router and vcr?